src/hardware/addon-protocols/rs232-usb.md

```diff
@@ -231,6 +231,19 @@ Issued by the GO device on receipt of Binary Data of 256 bytes or more from the
 | Checksum | 2 | 3+x |
 | ETX (0x03) | 1 | 5+x |
 
+### Msg Type 0x26: Protobuf data packet
+
+Issued by the GO device in response to Msg Type 0x8C.Information published by the GO device.
```

Comment:
Space missing after 0x8C. Could you please elaborate on what information is published by the GO device.